src/components/OutboundMessagePanel/OutboundMessagePanel.Container.js

```diff
@@ -12,16 +12,13 @@ import {
   Separator,
   Box,
 } from "@twilio-paste/core";
-
 import {
   Container,
   StyledSidePanel,
   DialerContainer,
   MessageContainer,
   SendMessageContainer,
   MessageTypeContainer,
-  OfflineContainer,
```

Comment:
Did the correct version get checked in or am I missing something? Aren't these required? thanks

Reply:
I just deployed this plugin using the updates that @cullenwatson made on this PR - those import do need to be added back in @dahamilton-twilio